src/Microsoft.AspNetCore.Server.Kestrel/Http/FrameResponseStream.cs

```diff
@@ -134,14 +134,14 @@ public void Abort()
             }
         }
 
-        public void Initialize(FrameContext context)
+        public void Initialize(IFrameControl context)
         {
-            _context = context;
```

Comment:
We could do the same rename in MessageBody.cs and TestInput.cs (though the test stuff is less important to me).